README.md

```diff
@@ -1,12 +1,14 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I practiced: responsive images, creating a form and accordion 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+This time I started coding the elements without first making a layout. That took me more time to fix in the end. Next time and from now on I will start with the lay out. I had issues with the image as backgorund image. Some confusion around the requirements of the project but it got clarified. I wanted to be able to target with CSS each image within the <picture> tag. I asked in other forums as well and tried with ID class but it didn't work for me. I was running out of time so chose to use media queries as I did last week. I want to experiment more with this pciture tag and also a responsive video would be next step for me.
```

Comment:
Nice to read your reflections for this project. I know you guys have limited time for each project so I know it can be a bit stressful when trying out new stuff - just know that you don't have to get it right now even if it's frustrating. You've got plenty of time to try stuff out in the bootcamp, and also when you're working as a developer. Stay curious! 🌟